### PR TITLE
RNW Support Policy Doc

### PR DESCRIPTION
**Pull request by jonthysell**
This PR adds a new support page, which documents and details the official support policy for RNW.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows-samples/pull/709)

### CODE REVIEW

**Comment by jonthysell** [comment]
I've added a markdown version of the file so it's easier to review.

File: website/pages/en/support.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+See below for further details on each support phase. For further details on the release process, see [RNW Versioning and Release Process](https://github.com/microsoft/react-native-windows/wiki/Versioning-and-Release-Process).
+
+## Support Phases
+
+### Active Support
+
+A stable version of RNW enters *Active Support* status as soon as the first official, non-preview, package of that stable version is published to NPM. This support continues with each patch release within that stable version.
+
+It is expected that the stable release branch is "stable" and will not take breaking API or template changes (relative to the first official release in that branch).
```
Comment (asklar) [inline website/pages/en/support.js:56]: project template? react-native-windows-init CLI project template?

File: website/pages/en/support.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+The RNW Team commits to providing support for fixing issues that do not require breaking changes, including but not limited to:
```
Comment (asklar) [inline website/pages/en/support.js:58]: During this window, ...

File: website/pages/en/support.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+- Security and/or legal issues
+- Build and/or tooling issues
+- Critical functional issues
+- Non-critical functional issues
```
Comment (asklar) [inline website/pages/en/support.js:63]: Non-critical -> Important ?
we may still not fix all issues that are non-critical :)

File: website/pages/en/support.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+### Active Support
+
+A stable version of RNW enters *Active Support* status as soon as the first official, non-preview, package of that stable version is published to NPM. This support continues with each patch release within that stable version.
+
+It is expected that the stable release branch is "stable" and will not take breaking API or template changes (relative to the first official release in that branch).
+
+The RNW Team commits to providing support for fixing issues that do not require breaking changes, including but not limited to:
+
+- Security and/or legal issues
+- Build and/or tooling issues
```
Comment (asklar) [inline website/pages/en/support.js:61]: do we want to mention "regressions in" here? 
e.g. VS2022/vc143 support is a build/tooling issue, but it's never worked before.

File: website/pages/en/support.js (hunk 3 above)
Comment (jonthysell) [reply website/pages/en/support.js:63]: Do you think maybe we need some "at their discretion?"

File: website/pages/en/support.js (hunk 4 above)
Comment (asklar) [reply website/pages/en/support.js:61]: mmm "precedent" :) I'd word all of these as aspirational and non-binding

File: website/pages/en/support.js [hunk 5]
```diff
@@ -97,7 +96,7 @@ A stable version of RNW enters *Preview Support* as soon as the first preview pa
 
 It is expected that the stable release branch in preview is "stabilizing," and it is the last chance to make large and/or breaking changes to that branch.
 
-The RNW Team commits to providing the same support as in [Active Support](#active-support), but breaking API and/or template changes may be allowed.
+While in *Preview Support*, the RNW Team commits to providing this stable version the equivalent of [Active Support](#active-support), but breaking API and/or template changes may be allowed.
```
Comment (asklar) [inline website/pages/en/support.js:99]: would recommend removing "commit" from this doc and say "strives to" instead.

**Review by asklar — APPROVED** [review]

**Review by marlenecota — APPROVED** [review]

**Pull request merged**